### PR TITLE
GPC-194: Setup prometheus alerts manager

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: terraform/modules/data-share-service/prometheus.tf [hunk 1]
```diff
@@ -11,11 +11,47 @@ resource "random_password" "prometheus_password" {
...
+      "aps:GetMetricMetadata",
+      "aps:GetSeries",
+      "aps:QueryMetrics",
+      "aps:ListRules",
+      "aps:ListAlertManagerSilences",
+      "aps:ListAlertManagerAlerts",
+      "aps:GetAlertManagerStatus",
+      "aps:ListAlertManagerAlertGroups",
+      "aps:PutAlertManagerSilences",
+      "aps:DeleteAlertManagerSilences",
```
Comment (ethanmills) [inline terraform/modules/data-share-service/prometheus.tf:27]: Does this mean that we _can_ set up AMP alert manager as a datasource in grafana?

File: terraform/modules/data-share-service/prometheus.tf [hunk 2]
```diff
@@ -11,11 +11,47 @@ resource "random_password" "prometheus_password" {
...
+resource "aws_prometheus_alert_manager_definition" "prometheus" {
+  workspace_id = aws_prometheus_workspace.prometheus.id
+  definition   = <<EOF
+alertmanager_config: |
+  route:
+    receiver: 'default'
+  receivers:
+    - name: 'default'
+      sns_configs:
+      - topic_arn: ${aws_sns_topic.sns_alarm_topic.arn}
```
Comment (ethanmills) [inline terraform/modules/data-share-service/prometheus.tf:53]: Is this the topic we've subscribed the email address to?

**Review by ethanmills — APPROVED** [review]

File: terraform/modules/data-share-service/prometheus.tf (hunk 1 above)
Comment (oskarwilliams378) [reply terraform/modules/data-share-service/prometheus.tf:27]: Yep :)

File: terraform/modules/data-share-service/prometheus.tf (hunk 2 above)
Comment (oskarwilliams378) [reply terraform/modules/data-share-service/prometheus.tf:53]: Yep :)

**Pull request merged**